tests/h/streamer/filter_test.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import unicode_literals
+
+from h.streamer.filter import FilterHandler
+
+
+class TestFilterHandler(object):
```

Comment:
These three tests cover the different ways that the client actively uses the real-time API. Since this is an undocumented API, I'm treating the client as the only consumer that we need to worry about.